.github/workflows/ci.yml

```diff
@@ -0,0 +1,84 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    # Run at 18:00 UTC every day
+    - cron: '0 18 * * *'
```

Comment:
Is this the link-check cronjob?

Reply:
From what I understand, all the steps in the file are executed when master or a PR is pushed to, and at a regular interval (cron job). Is that correct?

Reply:
I think you're partially correct, but there are several instances of https://github.com/rust-lang/rustc-dev-guide/pull/1073#discussion_r585800357.

Reply:
> Is this the link-check cronjob?

Yeah, it's the same as what we have on travis.

![image](https://user-images.githubusercontent.com/25030997/109706764-b08d3d80-7bdc-11eb-8ffb-9d24f2c259a0.png)

Reply:
Is there a reason to not have this in a separate yml file inside of workflows?

Reply:
> Is there a reason to not have this in a separate yml file inside of workflows?

I don't think it's worth having duplicate workflows with multiple files.